docs/content/guides/3.performance/2.core-web-vitals.md

```diff
@@ -0,0 +1,174 @@
+
+::code-group
+```html[Next.js]
+https://nextjs.org/docs/pages/api-reference/components/image#priority
+<Image
+  src="/image.png"
+  priority={true}
+/>
+```
+```html[Nuxt]
+https://image.nuxt.com/usage/nuxt-img#preload
+<NuxtImg 
+  src="/image.png" 
+  preload 
+/>
+```
+```html[Other]
+https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/rel/prefetch
+<head>
+  <link rel="prefetch" href="/image.png" fetchpriority="high" as="image" />
+</head>
+```
+::
+
+### Making Resources Smaller and Faster
+
+If you are already preloading resources, you can also improve LCP by making the resources load faster. This will help the browser load and paint the content faster.
+
+A few examples of this are:
+
+- Using the proper size image and not unnecessarily using a larger image
+- Using modern image formats
+- Reducing the web font size
+- Using a CDN to reduce the request times
+
+### Identifying the Largest Contentful Paint
+
+The easiest way to learn which element is the Largest Contentful Paint is using [PageSpeed Insights](https://pagespeed.web.dev/) page. After generating the report, the LCP metric and what element it relies on is displayed under the "Largest Contentful Paint element" diagnostic.
+
+Another option is to run Lighthouse in Chrome DevTools, described on [Google Developers](https://developers.google.com/web/tools/lighthouse#devtools) website.
+
+**See More**:
+- [Google's Guide on Optimizing LCP](https://web.dev/articles/optimize-lcp)
+
+## Interaction to Next Paint (INP)
+
+[Interaction to Next Paint (INP)](https://web.dev/articles/inp) measures how long a browser's paint cycle is blocked after a user interacts with a page. 
+
+Three areas where INP can be improved are:
+
+1. Input Delay - the time it takes for the browser to process user input and trigger event callbacks
+2. Event Callbacks - the time it takes for the browser to execute event callbacks after user input
+3. Rendering - the time it takes for the browser to render the page after user input
+
+We recommend reading Google's [Guide on Optimizing INP](https://web.dev/articles/inp) to learn more about how INP is calculated and some JavaScript techniques that can help improve it.
+
+For rendering, some common issues that can cause poor INP are:
+
+1. Complex CSS selectors - when a selector is too complex, the browser will have to parse and calculate the selector's specificity, which can slow down the rendering process after an interaction
+2. Large layout shifts - if an interaction causes a large layout shift, the browser will have to recalculate a large layout and then repaint the entire page, which can slow down the rendering process
+3. Large DOMs - the larger the DOM, the more expensive any rendering and updating operations will be
+
+
+**See More**:
```

Comment:
suggestion: I think we could also mention debugbear inp debugger here https://www.debugbear.com/inp-debugger